docs/integration-test-helpers/masonry/MasonryContainer.js

```diff
@@ -304,14 +366,16 @@ export default class MasonryContainer extends Component<Props, State> {
     }
 
     if (noScroll) {
-      dynamicGridProps.scrollContainer = () => undefined;
+      dynamicGridProps.scrollContainer = undefined;
     } else if (hasScrollContainer) {
-      dynamicGridProps.scrollContainer = () =>
-        typeof document === 'undefined'
-          ? undefined
-          : document.querySelector('[data-scroll-container]');
+      const query = document.querySelector('[data-scroll-container]');
+      if (typeof document === 'undefined' || query === null) {
```

Comment:
`query === null` is a runtime change — why is it needed?

Reply:
This is from previous attempts to make Flow be happy with querySelector being able to return null and         `dynamicGridProps.scrollContainer = () => query;` having to return HTMLElement

Reply:
Makes sense!